sw/device/tests/crypto/cryptotest/firmware/extclk_sca_fi.c

```diff
@@ -0,0 +1,76 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/tests/crypto/cryptotest/firmware/extclk_sca_fi.h"
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/dif/dif_clkmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/ottf_test_config.h"
+#include "sw/device/lib/testing/test_framework/ujson_ottf.h"
+#include "sw/device/lib/ujson/ujson.h"
+#include "sw/device/tests/crypto/cryptotest/firmware/status.h"
+#include "sw/device/tests/crypto/cryptotest/json/extclk_sca_fi_commands.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+static dif_clkmgr_t clkmgr;
+
+/**
+ * Extclk Configure command handler.
+ *
+ * Allows to configure the external clock.
+ * The uJSON data contains:
+ *  - sel: External clock on or off.
+ *  - hi_speed_sel: Nominal or low-speed external clock.
+ *
+ * @param uj The received uJSON data.
```

Comment:
I think what you mean is:
```suggestion
 * @param uj An initialized uJSON context.
```
?